app/scripts/controllers/rif/notifier/operations.js

```diff
@@ -71,4 +71,23 @@ export class NotifierOperations {
         }).catch(err => reject(err));
     });
   }
+  getRnsEvents (apiKey, nodehash = '', eventName = '') {
+    return new Promise(async (resolve, reject) => {
+      const notifierEndpoint = this.configurationProvider.getConfigurationObject().notifier.availableNodes[0];
+      const endpointParams = (nodehash || eventName ? '?' : '') + (nodehash ? 'nodehash=' + nodehash + (eventName ? '&' : '') : '') + (eventName ? 'eventName=' + eventName : '');
```

Comment:
What about move this to a function and construct the endpointParams in different lines? because its really difficult to understand the ternary ifs inside the concatenations

Reply:
There i put this in 3 lines with comments